src/contexts/UserContext.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react'
+
+const Context = React.createContext({})
+
+export function UserContextProvider ({children}) {
+  const [token, setToken] = useState(
+    () => JSON.parse(window.localStorage.getItem('token'))
+  )
+
+  return <Context.Provider value={{token, setToken}}>
+    {children}
+  </Context.Provider>
+}
+
+export default Context
```

Comment:
I would create a hook to avoid export the context and call it every time you need it with something like this;
`export const useUserContext = useContext(Context); `

and export as a default the `UserContextProvider`, no the `Context`